src/bootstrap/datatable.html

```diff
@@ -52,7 +52,7 @@
 
         <!-- Columns -->
         <td repeat.for="columnLabel of columnLabels">
-          <a click.delegate="selected(row)">${displayValue(row, columnLabel.column) | convertManager: columnLabel.converter}</a>
+          <a click.delegate="selected(row)" innerhtml.bind="displayValue(row, columnLabel.column) | convertManager: columnLabel.converter"></a>
```

Comment:
We want to be able to use HTML tags in our column. In our case we need it to show barcodes.